quarkus/config-api/src/main/java/org/keycloak/config/OpenApiSwaggerOptions.java

```diff
@@ -0,0 +1,11 @@
+package org.keycloak.config;
+
+public class OpenApiSwaggerOptions {
+
+    public static final Option<Boolean> OPENAPI_SWAGGER_ENABLED = new OptionBuilder<>("openapi-enabled", Boolean.class)
```

Comment:
@robson90 You will need to use a mapper to map this option to a Quarkus option, see e.g.:
https://github.com/keycloak/keycloak/blob/160ec6ba0d9171bb0dc4f882126f55128eb486e4/quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/HttpPropertyMappers.java#L67